Sensus.Shared.Tests/Shared/ProtocolTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using Sensus.Service.iOS.Context;
+using Sensus.Service.Tools.Context;
+using SensusService;
+
+namespace Sensus.Tests.Local.Shared
+{
+    [TestFixture]
+    public class ProtocolTests
+    {
+        #region Fields
+        private JsonSerializerSettings _jsonSerializerSettings;
+        #endregion
+
+        #region SetUp
+        [TestFixtureSetUp]
```

Comment:
Deprecated? Also, should I make those namespace changes now that we have a project structure?